include/HumanView.h

```diff
@@ -29,13 +40,24 @@ class HumanView : public View {
         sf::Keyboard::Key right;
         sf::Keyboard::Key up;
         sf::Keyboard::Key shoot;
+        int waitingForKey;
+        int currentLevel;
+        int dialogueStage;
+        void drawChadGirlBox();
+        float startTime;
+        float currTime;
+        bool first;
+        std::string response;
+        std::string preference;
+        int levelsWon;
     
     public: 
-        HumanView(sf::RenderWindow& App, GameLogic& logic, sf::View& pView): View( logic ), display(App), view(pView) {};
+        HumanView(sf::RenderWindow& App, GameLogic& logic, sf::View& pView, Strings& s): View( logic ), display(App), view(pView),  strings(s) {};
         
         void init();
 
         void update(float time);
+        void checkKeyPressed(sf::Keyboard::Key key);
```

Comment:
Change function name to specify that it checks the key pressed for rebinding the controls because this is too vague and general